index.js

```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
     })
 }
 
+var getAccountsBelowBal = (db, upperBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$lt: upperBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
+        callback(results)
+    })
+}
+
+var getAccountsAboveBal = (db, lowerBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$gt: lowerBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
+        callback(results)
```

Comment:
```suggestion
        callback(docs)
```

Reply:
Makes more sense to leave it as it is if I'm keeping the foreach from above